checker/main.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+
+	"github.com/systemboot/systemboot/pkg/checker"
+)
+
+var configFile string
+
+func init(){
```

Comment:
nit: I have a strong preference for minimizing the use of `init` functions (because of how they could be used to introduce implicit behaviours). In this case it's not even necessary - you can just declare it as
```
var configFile = flag.String("c", "", "Configuration file that defines checks and remediations")
```

Note that I also suggested to use "-c" instead of "-config-file" so we minimize the CLI verbosity